src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/ExternalLoginFailure.cshtml

```diff
@@ -1,8 +1,7 @@
-﻿@{
+@{
     ViewBag.Title = "Błąd logowania";
 }
 
-<hgroup>
+
```

Comment:
have you checked if it looks ok?

Reply:
Sorry perhaps I should use header tag. Am I right?

Reply:
I replaced it by <div> tag. Is it correct now?

Reply:
I have no idea :p Have checked the view?

Reply:
@Kamil4921 ?